code_examples/sdk_examples/src/workshop/attester/generateDid.ts

```diff
@@ -12,40 +9,33 @@ export async function createFullDid(
   fullDid: Kilt.DidDocument
 }> {
   const api = Kilt.ConfigService.get('api')
-
-  const mnemonic = Kilt.Utils.Crypto.mnemonicGenerate()
-  const {
-    authentication,
-    keyAgreement,
-    assertionMethod,
-    capabilityDelegation
-  } = generateKeypairs(mnemonic)
-  // Get tx that will create the DID on chain and DID-URI that can be used to resolve the DID Document.
-  const fullDidCreationTx = await Kilt.Did.getStoreTx(
-    {
-      authentication: [authentication],
-      keyAgreement: [keyAgreement],
-      assertionMethod: [assertionMethod],
-      capabilityDelegation: [capabilityDelegation]
-    },
-    submitterAccount.address,
-    async ({ data }) => ({
-      signature: authentication.sign(data),
-      keyType: authentication.type
-    })
-  )
-
-  await Kilt.Blockchain.signAndSubmitTx(fullDidCreationTx, submitterAccount)
-
-  const didUri = Kilt.Did.getFullDidUriFromKey(authentication)
-  const encodedFullDid = await api.call.did.query(Kilt.Did.toChain(didUri))
-  const { document } = Kilt.Did.linkedInfoFromChain(encodedFullDid)
-
-  if (!document) {
+  const mnemonic = process.env.ATTESTER_ACCOUNT_MNEMONIC as string
+  const { type, publicKey } = submitterAccount
+
+  const txs = [
+    api.tx.did.createFromAccount({ [type as 'ed25519']: publicKey }),
```

Comment:
we could also get around this type forcing by changing the function parameter on line 6 to 
`submitterAccount: Kilt.KiltKeyringPair & { type: 'ed25519' }`